Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN apt update \
   && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
   && truncate -s 0 /var/log/*log
 
+RUN pip3 install pyasn1 
```

Comment:
You need to modifiy the setup.py

Reply:
also @sven-hash can you fix version to pyasn1==0.4.8 in setup.py ? Thank you

Reply:
Done and removed in the Dockerfile